text/0000-docs-information-architecture

```diff
@@ -0,0 +1,154 @@
+
+Tutorials
+- Quick Start
+- Build and deploy a site
+    - Introduction
+    - 0. Setting up the development environment
+    - 1. Creating pages
+    - 2. Adding styles
+    - 3. Using layouts to build reusable site sections
+    - 4. Deploying a site live online (debating whether to keep this inside part 1, where it currently is)
+- Build a blog with GraphQL
+    - 5. Querying data for your blog
+    - 6. Source plugins and rendering queried data
+    - 7. Transformer plugins and transforming data into site content
+    - 8. Programmatically creating pages from data
+- Adding images to your site (or "Adding svg or blur-up effect to images)*
+- Creating your own plugin (tutorial for creating a plugin we've already vetted as easy to create)*
+- E-commerce*
+- User authentication*
+
+Reference docs
+- How Gatsby works
+    - The Gatsby core philosophy*
+    - Basic structure of a Gatsby site (includes config.js)*
+    - How Gatsby works with GraphQL (includes Node interface and how GraphQL works)*
+    - How Gatsby works with React (API specification)*
+    - How Gatsby works with Redux (bound action creators)*
+    - How Gatsby works with plugins
+    - PRPL pattern
+    - Lifecycle APIs
+- API references
+    - @jlengstorf this idea is from you, don't know how to fill this in...*
+    - API specification
+    - Gatsby Browser APIs
+    - Gatsby Node APIs
+    - Gatsby SSR APIs 
+
+- Gatsby CLI commands
+- Basic tasks
+    - Setting up dev environment   
+        - Gatsby on Windows
+    - Customizing site structure
+        - html.js
+        - webpack config
+        - Environment variables (right place for this??)
+    - Adding content 
+        - Images
+        - Markdown pages
+        - adding tags and categories to blogposts
+        - adding a 404 page
+    - Querying for data with GraphQL
+    - Debugging
+        - Debugging HTML builds
+        - Debugging Replace Renderer API
+    - Adding dynamic functionality
+        - Building apps with Gatsby
+        - E-commerce
+        - authentication
+        - dynamically rendered navigation*
+    - Deploying
+    - Hosting
+        - Path Prefix
+    - Performance
+        - Search Engine Optimization (SEO)
+        - Browser 
+        - Setting up proper HTTP caching
+- Migration Guides
+    - v0 to v1
+    - v1 to v2*
+    - v0 to v2 (good idea??)*
+- Gatsby components and libraries
+    - Glamor
+    - Styled components
+    - gatsby-image
+
+
+Plugins
+- Plugin Authoring Guides
+    - How Gatsby works with plugins
+    - Plugin library
+    - When to create your own plugin?*
+    - Source plugins
+    - Transformer plugins
+    - All other plugins
+    - How to get your plugin to show up in the Plugin Library*
+
+
+Contributing
+- Welcome to Gatsby's open source community!
+- Code of Conduct
+- How to Contribute
+    - List of ways to contribute*
+    - How to file an issue
+    - How to submit a PR*
```

Comment:
Also something about speaking about Gatsby at a local event?

Reply:
(or writing a blog post or even a tweet!)

Reply:
Yes, those are actually under the templates section. And the list of ways to contribute already includes those ideas, although the doc is a bit too long. I wonder if it makes sense to put the templates under this section instead of their own section, though